lib/improver/utilities/load.py

```diff
@@ -38,6 +38,39 @@
 from improver.utilities.cube_manipulation import (
     enforce_coordinate_ordering, merge_cubes)
 
+GRID_ID_LOOKUP = {'enukx_standard_v1': {'mosg__grid_type': 'standard',
+                                        'mosg__model_configuration': 'uk_ens',
+                                        'mosg__grid_domain': 'uk_extended',
+                                        'mosg__grid_version': '1.1.0'},
+                  'engl_standard_v1': {'mosg__grid_type': 'standard',
+                                       'mosg__model_configuration': 'gl_ens',
+                                       'mosg__grid_domain': 'global',
+                                       'mosg__grid_version': '1.1.0'},
+                  'ukvx_standard_v1': {'mosg__grid_type': 'standard',
+                                       'mosg__model_configuration': 'uk_det',
+                                       'mosg__grid_domain': 'uk_extended',
+                                       'mosg__grid_version': '1.1.0'},
+                  'glm_standard_v1': {'mosg__grid_type': 'standard',
+                                      'mosg__model_configuration': 'gl_det',
+                                      'mosg__grid_domain': 'global',
+                                      'mosg__grid_version': '1.1.0'},
+                  }
+
+
+def _handle_old_versions(cube):
```

Comment:
@MoseleyS I was wondering how you would see this being extending to handle e.g. StaGE version 1.3.0 and 1.4.0? Would this function end up being quite complicated? It's not great having to define the contents of `GRID_ID_LOOKUP` here, as this is essentially bespoke metadata from a different system. 

Could this not be handled specifically within the get_level_1 step, similarly to IMPRO-835, so that the StaGE to IMPROVER conversion is limited the get_level_1 step, rather than being in the IMPROVER codebase?